tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AWS/AwsSnsTests.cs

```diff
@@ -53,10 +53,10 @@ public async Task SubmitsTraces(string packageVersion, string metadataSchemaVers
             using (RunSampleAndWaitForExit(agent, packageVersion: packageVersion))
             {
 #if NETFRAMEWORK
-                var expectedCount = 3;
+                var expectedCount = 8;
```

Comment:
where does 8 come from?

Reply:
Number of expected spans for the NET Framework. Double as compared to NET Core.